### DISCUSSION TITLE
Launch options / Launch parameters

### DISCUSSION THREAD

**Discussion by PostalVoid** [Eugeny/tabby#9073] [Q&A]
Keep in mind that I'm running on linux, so this question might not pertain 1:1 for what windows users are looking for.

I've looking around for a way to launch tabby with parameters. For example, if I wanna have open a new tab in the current directory of my file manager, doing `konsole --new-tab --workdir %f` with konsole and thunar does the trick. However, I haven't found a resource to check launch options. There's no man pages, and trying `tabby -h` or `tabby --help` only opens another instance of tabby rather than provide further info on flags I can launch the program with. I've tried looking for any info on launch options, and I've come up with nothing.

**Comment by stibinator** [comment]
Ditto. I want to use it as my default terminal emulator, but I can't add actions to the desktop environment's context menus like I can with any other terminal. For example, I'd like to be able to open a file in nvim in Tabby.
